modules/core/src/main/scala/com.snowplowanalytics.snowplow.bigquery/processing/BigQuerySchemaUtils.scala

```diff
@@ -16,7 +16,7 @@ import scala.jdk.CollectionConverters._
 
 object BigQuerySchemaUtils {
 
-  def alterTableRequired(tableDescriptor: Descriptors.Descriptor, ddlFields: List[Field]): Boolean =
+  def alterTableRequired(tableDescriptor: Descriptors.Descriptor, ddlFields: Seq[Field]): Boolean =
```

Comment:
You changed this from `List` to `Seq`.  But I tried changing it back to `List` again and it still compiles.

In places where we consistently pass in a List, I would prefer to be explicit about it in the method signature.

Reply:
Oh right, it compile because I added `.toList` [here](https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/7ac7df386bb6ea0071ad8a02a0521fc04521a86d/modules/core/src/main/scala/com.snowplowanalytics.snowplow.bigquery/processing/Processing.scala#L300) so I convert `Vector` to `List`. Then I changed arg to accept `Seq` so we don't have to convert. But forgot to remove `.toList`!